scripts/terminalserver/terminalserver.jl

```diff
@@ -1,23 +1,29 @@
-# this script basially only handles `Base.ARGS`
+# this script basially only handles `ARGS`
+
 
 Base.push!(LOAD_PATH, joinpath(@__DIR__, "..", "packages"))
 using VSCodeServer
 pop!(LOAD_PATH)
 
-# load Revise ?
-if "USE_REVISE" in Base.ARGS
-    try
-        @eval using Revise
-        Revise.async_steal_repl_backend()
-    catch err
+let
+    args = [popfirst!(Base.ARGS) for _ in 1:5]
+    # load Revise ?
+    if "USE_REVISE=true" in args
+        try
+            @static if VERSION ≥ v"1.5"
```

Comment:
I see the logic, but doesn't this depend on the version of Revise? If I'm on Julia 1.5 and have an older version of Revise.jl installed, we still need the `async_steal_repl_backend` call, right?

Reply:
yes, we can have something like
```julia
using Pkg

function revise_installed()
    function check(deps)
        return filter(collect(values(deps))) do pkg_info
            return pkg_info.name == "Revise" && 
                pkg_info.version ≥ v"2.7"
        end |> !isempty
    end

    # current
    check(Pkg.dependencies()) && return true

    # fallback
    p = normpath(Pkg.envdir(), string('v', VERSION.major, '.', VERSION.minor), "Project.toml")
    env = Pkg.Types.EnvCache(p)
    ctx = Pkg.Types.Context(; env)
    check(Pkg.dependencies(ctx)) && return true

    return false
end
```

but why I didn't include this into the patch was it may slow down our startup time.

Reply:
I'll try to revive my "ship Revise.jl out of the box" PR, so I think this right now is a great fix for the moment.